doc/source/examples/dem/bouncing-particle/bouncing-particle.rst

```diff
@@ -0,0 +1,168 @@
+
+.. code-block:: text
+
+    python3 Lethe_case_generator.py 5000000
+
+Where ``5000000`` represent the normal spring constant that is wish to be used for the simulations. This code will generate 6 files named ``bouncing_particle_XX.prm``, where ``XX`` represent the value of the restitution coefficient used in it.
+
+Running the simulation
+----------------------
+Once all 6 parameter file are created, the simulation can be launched one by one using the following line (parallel mode is not recommend since there is only one particle):
+
+.. code-block:: text
+
+  dem_3d bouncing_particle_XX.prm
+
+Every simulation takes less than 30 seconds to run. A folder named according to the restitution coefficient used will be generated (``/out_xx``).
+
+Post-processing
+---------------
+A Python post-processing code called ``Bouncing_particle_post_processing.py`` is provided with this example. It is used to compare the height reached by the particle at each bounces with the analytical solution of a hard sphere bouncing on a flat plane. This analytical solution consider instantaneous collision between the particle and the wall, thus the maximum height of each bounce can be express by the following expression:
+
+.. math::
+        h_{k}^{max} = (h_0 - r_p)e^{2k} + rp
+
+with :math:`k` represent the :math:`k^{th}` bounce, :math:`h_0` the starting height and :math:`r_p` the radius of the particle.
+
+Once the 6 simulations have been run, used the following line in your command line to run the post-processing code :
+
+.. code-block:: text
+
+  python3 Bouncing_particle_post_processing.py
+
+A figure will then appear comparing the expected values results with the analytical solution.
+
+Results and discussion
+-----------------------
+Animation of a bouncing particle with different restitution coefficient (:math:`K_n` = 5E6 N/m):
+
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/B1n776jHdoo" frameborder="0" allowfullscreen></iframe>
+
+It can be seen that the particle with a restitution coefficient (:math:`e`) of 1.0 doesn't have any change in its rebounds. The other particles show a reduction in rebound height that seems to respect the analytical solution express earlier.
+
+Now, using the post-processing code, it is possible to compare the effect of the normal spring constant of the conservation of the kinetic energy during the collision.
+
+.. list-table::
+
+    * - .. figure:: images/kn5E4.png
+           :width: 260
+           :alt: Mesh
+           :align: center
+
+           Spring constant equal to 5E4.
```

Comment:
The images are nice but it's not clear. You don't have symbols for the analytical solution and for the simulation results. Can you try to make a figure a bit clearer so we at least recognize that there are two symbols (or use a line for simulationa nd symbols for the analytical solution)
Must be clearer. It's very good, but requires a bit more work.

Reply:
I have symbol for the analytical solutions, right now it's small cross, but I understand that's not visible with the resolution used in the html. I'll change the marker types so that it's clearer.

Reply:
I'll ajust the current label in the legend so that we know ithey are the analytical.

Should I add the "numerical" label to the legend? I'm scare that there will be to many label in it...